cmd/duffle/export.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"io"
 
+	builder2 "github.com/deislabs/duffle/pkg/imagestore/builder"
```

Comment:
nit: Why the name `builder2`?

Reply:
That was unintentional and due to a refactoring without sufficient eye-balling afterwards. Will fix.

Reply:
Actually, it has already gone!